wgpu/tests/occlusion_query/mod.rs

```diff
@@ -0,0 +1,136 @@
+use crate::common::{initialize_test, TestParameters};
+use std::borrow::Cow;
+use wasm_bindgen_test::wasm_bindgen_test;
+
+#[test]
+#[wasm_bindgen_test]
```

Comment:
Will do a more thorough review after the deno part is reviewed, but this was exactly what I wanted - thank you!